newm-server/src/main/kotlin/io/newm/server/database/migration/V68__CreateMintingStatusHistory.kt

```diff
@@ -0,0 +1,24 @@
+package io.newm.server.database.migration
+
+import org.flywaydb.core.api.migration.BaseJavaMigration
+import org.flywaydb.core.api.migration.Context
+import org.jetbrains.exposed.sql.transactions.transaction
+
+@Suppress("unused")
+class V68__CreateMintingStatusHistory : BaseJavaMigration() {
+    override fun migrate(context: Context?) {
+        transaction {
+            exec(
+                """
+                CREATE TABLE IF NOT EXISTS minting_status_history (
+                    id uuid PRIMARY KEY,
+                    time_stamp timestamp without time zone NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
Let's call the `time_stamp -> created_at` to follow the pattern of all the other tables.